tests/PKSim.Tests/IntegrationTests/FlatParameterInContainsRepositorySpecs.cs

```diff
@@ -31,6 +33,11 @@ protected string ErrorMessageFor(IEnumerable<ParameterMetaData> parameters)
 
    public class When_checking_the_database_consistency_regarding_parameter_flags : concern_for_FlatParameterInContainsRepository
    {
+      private bool isOneOfReadOnlyAndIsInputParameters(string parameterName)
+      {
+         return parameterName.IsOneOf(CoreConstants.Parameters.IS_SMALL_MOLECULE, CoreConstants.Parameters.INPUT_DOSE);
+      }
```

Comment:
Those parameters which must be defined as input and are read-only because of the special handling